mongo/gridfs/bucket.go

```diff
@@ -381,14 +381,16 @@ func (b *Bucket) openDownloadStream(filter interface{}, opts ...*options.FindOpt
 	}
 
 	if foundFile.Length == 0 {
-		return newDownloadStream(nil, b.chunkSize, &foundFile), nil
+		return newDownloadStream(nil, foundFile.ChunkSize, &foundFile), nil
```

Comment:
I assume from this that foundFile will always have a chunkSize?

Reply:
Yeah, it's a required field. I updated the code to add a new error if it's missing.